### PR TITLE
OSDOCS-3606: Add release note for IPv6 primary family

### PR DESCRIPTION
**Pull request by jboxman-rh**
- https://issues.redhat.com/browse/OSDOCS-3606

Version(s): 4.13

Issue:

Link to docs preview:

- (vSphere) https://57840--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-dual-stack-vmware-vsphere-clusters
- (Bare metal) https://57840--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-ipv6-primary-address-family-bare-metal-dual-stack-clusters

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@cybertron, this is a first attempt a release note for this.

**Comment by jboxman-rh** [comment]
@sgoveas, I've created a few release notes for this as well.

**Comment by sgoveas** [comment]
@jboxman-rh, just want to confirm that the links in these notes would point to updates in #57903

**Comment by sgoveas** [comment]
Thanks, ack
/lgtm

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -256,6 +256,21 @@ For more information, see xref:../networking/openshift_sdn/migrate-to-openshift-
 
 The cluster network can be expanded to support the addition of nodes to the cluster. For example, if you deployed a cluster and specified `10.128.0.0/19` as the cluster network range and a host prefix of `23`, you are limited to 16 nodes. You can expand that to 510 nodes by changing the CIDR mask on a cluster to `/14`. For more information, see xref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range].
 
+[id="ocp-4-13-dual-stack-vmware-vsphere-clusters"]
+==== Dual-stack IPv4/IPv6 on VMware vSphere clusters
+
+On installer-provisioned vSphere clusters, you can use dual-stack networking with the IPv4 address family as the primary IP family, and the IPv6 address family as the secondary address family. For more information, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installation-configuration-parameters-network_installing-vsphere-installer-provisioned-network-customizations[Network configuration parameters].
```
Comment (mburke5678) [inline release_notes/ocp-4-13-release-notes.adoc:262]: Does this work (makes it a bit tighter):

s/IPv4 address family as the primary IP family, and the IPv6 address family as the secondary address family/IPv4 as the primary IP family, and the IPv6 as the secondary address family

**Comment by mburke5678** [comment]
@jboxman-rh One nit. Otherwise LGTM

**Comment by jboxman-rh** [comment]
@mburke5678, great, thanks!

**Pull request merged**